Extensions/EdFi.Ods.Extensions.TPDM/Artifacts/MsSql/Data/Security/1010-TPDM-ResourceClaims.sql

```diff
@@ -1987,6 +1987,40 @@ BEGIN
     INSERT INTO @claimIdStack (ResourceClaimId) VALUES (@claimId)
 
     -- Processing children of http://ed-fi.org/ods/identity/claims/domains/tpdm/students
+	----------------------------------------------------------------------------------------------------------------------------
```

Comment:
It looks like with these changes we have also removed quite a few resources. We've added one here that is missing but we should also remove the old ones. Whatever process was used to generate this file would probably make that pretty easy.

Reply:
ODS-5116 will address the removal, I was trying to avoid duplicate work if TPDM team can provide the claim setup according to their needs.